src/openrpc/user_grants.json

```diff
@@ -391,6 +391,14 @@
                         "minItems": 1
                     },
                     "required": true
+                },
+                {
+                    "name": "options",
+                    "summary": "Usergrant options",
+                    "schema": {
+                        "$ref": "#/components/schemas/UserGrantOption"
```

Comment:
We should pluralize `Options` to be consistent with the param name and other similar objects.

I"m also wondering if `RequestOptions` is a better name, since there might be other options objects in this module in the future.